platform/marvell/sonic-platform-nokia/7215/service/cpu_wdt.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=CPU WDT
-After=nokia-7215init.service
+After=nokia-7215init.service watchdog-control.service
```

Comment:
@Pavan-Nokia we don't need two watchdog services. Please override the watchdog-control.service completely

Reply:
@prgeor the watchcgdog-control.service is part of common code and is triggered by the docker. There is no good way to disable this service from a platform context.
The only way I could think of is to crudely delete the script so that the watchdog-control.service fails, which I believe is not a great solution. that is the reason I chose this approach.
I do not understand why watchdog-control.service was introduced by SONiC.

Reply:
@saiarcot895 do you have a better way to override watchdog-control.service by platform specific watchdog service?

Reply:
@saiarcot895 any feedback on this. I tried disabling the watchdog-control.service from the platform postinstall but it gets reenabled on next boot.
This is the only solution I could come up with that works reliably every time.

Reply:
You can use `/usr/local/lib/systemd/system/watchdog-control.service` to provide a platform-specific `watchdog-control.service` file. Specifically, systemctl treats files in `/usr/local/lib/systemd/system/` as higher priority than `/usr/lib/systemd/system/`, but lower than `/etc/systemd/system/` (which should still allow service mask/unmask to work).

Make sure that your platform-specific file has all of the ordering and conditions (Provides, After, WantedBy) of the original service file.

Reply:
@saiarcot895
If I understand what you are saying correctly, You are suggesting that I eliminate the cpu_wdt.service completely and run my script under watchdog-control.service. I do not think this is a good Idea as the behavior of these 2 services are intended to be totally different. The watchdog-control is designed as a one shot to turn off the watchdog here as what the intent of cpu_wdt is to enable the watchdog as early as we can and keep running.

The best approach I think would be to have a different service for the platform watchdog and to disable the watchdog-control for our platform. Changing the functionality of an existing service is not good

Reply:
I'm fine with that approach, as long as on this platform, only one of the two services can be started. This would mean masking the service early in the bootup (perhaps by adding a symlink at `/usr/lib/systemd/system/watchdog-control.service` to point to `/dev/null`?).

Reply:
@saiarcot895 Could you please review my final changes 
Thank you